.devcontainer/Dockerfile

```diff
@@ -1,11 +1,21 @@
-ARG VARIANT="20.18.1-bookworm"
+ARG VARIANT="20.18.2-bookworm"
 FROM node:${VARIANT}
 
+# Install some extras such as vim for interactive rebases. Also some
+# Cypress prerequisites for running in Debian containers:
+# https://docs.cypress.io/app/get-started/install-cypress#UbuntuDebian
+
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y \
-        # Cypress prerequisites for running in Debian containers:
-        # https://docs.cypress.io/app/get-started/install-cypress#UbuntuDebian
-        libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libnss3 libxss1 \
-        libasound2 libxtst6 xauth xvfb \
-        # For interactive git rebases
-        vim
+        libasound2 \
+        libgbm-dev \
+        libgtk-3-0 \
+        libgtk2.0-0 \
+        libnotify-dev \
+        libnss3 \
+        libxss1 \
+        libxtst6 \
+        vim \
+        xauth \
+        xvfb \
```

Comment:
Sonar wants the new packages to be in alphabetical order. While I'm usually 100% onboard with alphabetizing things, this does make it a bit more effort to check that we have all the Cypress prerequisites (in case those change).

Reply:
Since this is not a production image we could just ignore it in Sonar. Up to you... but I would be ok with ignoring this file.

Reply:
Maybe it's good to follow the Sonar recommendation - I don't think we'll be checking the Cypress prerequisites that often.